src/platform/forms/save-in-progress/FormSignInModal.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+
+class FormSignInModal extends React.Component {
+  handleSignIn = () => {
+    this.props.onClose();
+    this.props.onSignIn();
```

Comment:
This is a case where coupling the GA event with the action might be a problem... It seems like we might want to report a `login-link-restart-form` event here, but since it's opening the modal, by default, we would also be reporting a `login-link-clicked-cta` event.

Reply:
The question is whether we are okay with both events firing.

Reply:
This has been OK'd from a product perspective.